images/salt-master/Dockerfile

```diff
@@ -10,8 +10,8 @@ ARG TINI_VERSION=v0.18.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
 
-RUN gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
- && gpg --batch --verify /tini.asc /tini
+COPY source.key source.key
```

Comment:
shouldnt the image make target depends on this file too ? ;)

Reply:
Well... it doesn't _have_ to.

I guess it makes sense to have a more sensible error message than a gpg error message when building if the repo ever gets to a state where the key is removed, but on the other hand we have no way to "build" it anyway so there's little point to having it function as a dependency.

For comprehensiveness though, I'll add it.

Reply:
I meant in case you change the key file the container img wont get rebuilt